gen/dcos-config.yaml

```diff
@@ -985,23 +985,30 @@ package:
 
       ssl_certificate includes/snakeoil.crt;
       ssl_certificate_key includes/snakeoil.key;
-  - path: /etc/adminrouter-tls.conf
+  - path: /etc/adminrouter-tls-master.conf
     content: |
       # Ref: https://github.com/cloudflare/sslconfig/blob/master/conf
       # Modulo ChaCha20 cipher.
-      ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
+{% switch adminrouter_tls_cipher_override %}
+{% case "false" %}
+      ssl_ciphers EECDH+AES256:RSA+AES256:EECDH+AES128:RSA+AES128:EECDH+3DES:RSA+3DES:!MD5;
```

Comment:
I only now saw that [this commit|https://github.com/dcos/dcos/pull/2199/commits/63f60a3321175cf702ff919b44c480dbb7f1c0de] 'sneaks' in a change of the default cipher suite. We change from

> EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5

to 

> EECDH+AES256:RSA+AES256:EECDH+AES128:RSA+AES128:EECDH+3DES:RSA+3DES:!MD5

I do not see this discussed or motivated properly. It also renders the corresponding comment invalid (what we have so far is indeed the Cloudflare config from 2016 with the mentioned diff). @mhrabovcin I'd very much appreciate if you could revert that. That's a matter of following process.

CC @keith-mcclellan that's something to keep in mind for the future: let's treat independent what's independent :-). Overall, that minimizes risk and also smoothens integration.

Reply:
Reverted.